pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -95,6 +96,9 @@ func NewDefaultProvider(ctx context.Context, cache *cache.Cache, ec2api ec2iface
 		cm:                    pretty.NewChangeMonitor(),
 		KubeDNSIP:             kubeDNSIP,
 		ClusterEndpoint:       clusterEndpoint,
+		ClusterIPFamily: convertIPFamily(*lo.Must(eksapi.DescribeClusterWithContext(ctx, &eks.DescribeClusterInput{
```

Comment:
Can't you get this from `kubeDNSIP`?

Reply:
We had a offline discussion with @jonathan-innis and want to move away from deriving IPFamily from service IP.

Reply:
We already call describe cluster here: https://github.com/aws/karpenter-provider-aws/blob/b0a8f035d4cd4bc3a9315373a402b4ff43e563cd/pkg/operator/operator.go#L119, would be nice if we could just reuse that output rather than calling it multiple times in our startup logic.

Reply:
Let's me check if I can rename and reuse the func to return ipfamily as well.

Reply:
I see. Makes sense for self managed clusters.